common/djangoapps/util/db.py

```diff
@@ -231,3 +231,16 @@ def generate_int_id(minimum=0, maximum=MYSQL_MAX_INT, used_ids=None):
         cid = random.randint(minimum, maximum)
 
     return cid
+
+
+class MigrationModules(object):
```

Comment:
Nit: this should be named to make it clear what it is doing. Maybe ``NoOpMigrationModules`` or something? The current name sounds like this is a full implementation, not a testing-only one.